core-bundle/src/Controller/ContentElement/TableController.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Controller\ContentElement;
+
+use Contao\ContentModel;
+use Contao\CoreBundle\ServiceAnnotation\ContentElement;
+use Contao\CoreBundle\Twig\FragmentTemplate;
+use Contao\StringUtil;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * @ContentElement(category="texts")
+ */
+class TableController extends AbstractContentElementController
+{
+    protected function getResponse(FragmentTemplate $template, ContentModel $model, Request $request): Response
+    {
+        // Table fields
+        $rows = StringUtil::deserialize($model->tableitems, true);
+        $header = $model->thead && $rows ? array_shift($rows) : [];
+        $footer = $model->tfoot && $rows ? array_pop($rows) : [];
+
+        $template->set('header', $header);
+        $template->set('footer', $footer);
+        $template->set('rows', $rows);
+        $template->set('use_row_headers', (bool) $model->tleft);
+
+        $template->set('caption', $model->summary ?: null);
+
+        // Client side sorting
+        $sorting = !$header || !$model->sortable ? null : [
```

Comment:
```suggestion
        $sorting = (!$header || !$model->sortable) ? null : [
```

Not sure if the operator precedence is correct otherwise.

Reply:
`||` has a higher precedence than `? :`, so it should work, But I agree, that the parantheses make it more clear, so let's go with it. :+1:

Reply:
Or maybe not. The CS fixer (`UselessParenthesesSniff`) removed it again :smile: